app/controllers/publishers/vacancies_controller.rb

```diff
@@ -39,6 +37,13 @@ def review
     @vacancy = VacancyPresenter.new(vacancy)
   end
 
+  def live
+    return redirect_to organisation_job_review_path(vacancy.id), notice: t(".notice") unless vacancy.published?
+
+    @vacancy = VacancyPresenter.new(vacancy)
+    render "publishers/vacancies/_show"
```

Comment:
What differences are there between this view and the _real_ live listing that jobseekers see?

If it's very few, would it make more sense to use the same view for both, plus some conditionals like `unless publisher_signed_in?`? Because if we show publishers a page and call it "the live listing", they might share the link to that page (copying it from the address bar), instead of the link to the _real_ live listing as seen by jobseekers; and a jobseeker who follows that incorrect link will be required to authenticate as a publisher to see the listing.

Reply:
Very good point! In talks with Jesse to define what we want on that

Reply:
was this resolved? @cesidio @david-mears-dfe

Reply:
as per this conversation on slack:
https://ukgovernmentdfe.slack.com/archives/CG9A8H1HP/p1626161646232600
the link now points to the public live vacancy